src/main/bash/sdkman-init.sh

```diff
@@ -225,8 +225,9 @@ if [[ "$sdkman_auto_env" == "true" ]]; then
 
 			export SDKMAN_OLD_PWD="$PWD"
 		}
-
-		[[ -z "$PROMPT_COMMAND" ]] && PROMPT_COMMAND="sdkman_auto_env" || PROMPT_COMMAND="${PROMPT_COMMAND%\;};sdkman_auto_env"
+		
+		TRIMMED_PROMPT_COMMAND="${PROMPT_COMMAND%%*( )}"
```

Comment:
We normally use all uppercase var names only for environment variables.

So something like `trimmed_prompt_command` would be more fitting.

Reply:
Ok, tommorow i will change.